### DISCUSSION TITLE
Create Option text in InputPicker

### DISCUSSION THREAD

**Discussion by Abdullah1lb** [rsuite/rsuite#3660] [Q&A] [answered]
how can i change "Create Option" text in InputPicker to the custom text?

**Comment by simonguo** [comment, marked as answer]
You can override the global locale through the `locale` prop of the component.

```tsx
<InputPicker
  creatable
  locale={{ createOption: 'New Item: {0}' }}
/>
```

In addition, `CustomProvider` component can be used to override the global locale. https://rsuitejs.com/guide/i18n/

**Reply by Abdullah1lb** [reply]
Thank you!
